docs/commands/rpc-protocol.md

```diff
@@ -246,7 +246,7 @@ Returns the voting weight for **account**
 Returns how many RAW is owned and how many have not yet been received by **accounts list**  
 
 --8<-- "unconfirmed-information.md"
-    The pending balances are calculated from potentially unconfirmed blocks. Account balances are obtained from their frontiers. As long as you follow the [guidelines](/integration-guides/key-management/#transaction-order-and-correctness), you can rely on the **balance** for the purposes of creating transactions for these accounts.
+    The pending balances are calculated from potentially unconfirmed blocks. Account balances are obtained from their frontiers. Prefer using [account_info](#account_info) for the purposes of creating transactions.
```

Comment:
For the end maybe "An atomic [account_info](#account_info) RPC call is recommended for the purposes of creating transactions."